ts/components/session/settings/SessionSettings.tsx

```diff
@@ -441,7 +442,10 @@ export class SettingsView extends React.Component<SettingsViewProps, State> {
         id: 'message-ttl',
         title: window.i18n('messageTTL'),
         description: window.i18n('messageTTLSettingDescription'),
```

Comment:
just note that this setting is not used anywhere in our sending code at the moment and will need to be hooked up when we do enable it